types/rtlcss/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
       "module": "commonjs",
       "lib": [
-        "es6"
+        "es6",
+        "ES2018.Promise"
```

Comment:
Are we using this lib anywhere?

Reply:
Nope, it's required for DT linter only, and the 'tsconfig' is for DT consumption as well.
The problem is postcss 8 comes with modern language features, including 'finally' support for Promise, 
so as external lib, that is a way to suppress the TS error for extenral dependency.
```bash
Error: Errors in typescript@4.2 for external dependencies:
node_modules/postcss/lib/lazy-result.d.ts(60,28): error TS2339: Property 'finally' does not exist on type 'Promise<Result>'.
```